### ISSUE TITLE
[Combined Debt Portal] Prototype

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#35843]
## Issue Description
Veterans expect to find all money they owe to VA in one location on the site. Additionally, Congress has mandated that we provide a way for Veterans to access all their debts in one spot. As a result, we need to create a finalized prototype of the combined VBA + VHA debt portal.

This work should be completed once the wireframes have been reviewed and approved by stakeholders and early IA recommendations are incorporated. We should also kick off the collab cycle if we haven't already!

---
## Tasks
- [x] Create hi-fi prototypes of the combined debt portal

## Acceptance Criteria
- [x] Prototypes have been linked in this ticket

**Comment by rileyorr** [comment]
Added the designs to [this UXPin prototype](https://preview.uxpin.com/29cd8f525781fa231d16f5b50905ad15a99cbf01?_gl=1*1q65sth*_gcl_aw*R0NMLjE2NDM3MzgwNDMuRUFJYUlRb2JDaE1JcElULW5vbmY5UUlWTXRTekNoMk9aQUM4RUFBWUFTQUFFZ0tHQnZEX0J3RQ..#/pages/148307563/simulate/sitemap?mode=i). I'm going to review them again on Monday and make some adjustments; some of the pages have multiple options so I want to look at it again with fresh eyes. I'll also go to the design/research feedback office hours on Monday.

**Comment by rileyorr** [comment]
Incorporated feedback from the design/research office hours and the design feedback from the design intent. 

Main changes:
- The info alert notifying Veterans of additional debt types while on either the current debt page or current copay bill page seemed out of place within the context of the page. What we chatted about was putting that content redirecting the Veteran to the overview page in its own section that would display if the API call returned additional debt types. I've mocked that out on [the copay bill list page](https://preview.uxpin.com/29cd8f525781fa231d16f5b50905ad15a99cbf01?_gl=1*j83yn9*_gcl_aw*R0NMLjE2NDM3MzgwNDMuRUFJYUlRb2JDaE1JcElULW5vbmY5UUlWTXRTekNoMk9aQUM4RUFBWUFTQUFFZ0tHQnZEX0J3RQ..#/pages/148355892/simulate/sitemap?mode=i)
- I removed the back button that competed with the breadcrumbs as pointed out in the design intent
